docs/contents/user-docs/cleanup.md

```diff
@@ -0,0 +1,60 @@
+# Cleanup 
+
+Few custom resource definitions (CRDs) are common across services. If you have multiple controllers installed, you should not delete the common CRDs unless you are uninstalling all of the controllers.
+
+## Uninstall ACK service controllers and CRDs
+
+Use the following commands to uninstall ACK service controllers and delete CRDs:
+```bash
+export SERVICE=sagemaker
+export CHART_EXPORT_PATH=/tmp/chart
+
+# Uninstall the ACK service controller with Helm
+helm uninstall -n $ACK_K8S_NAMESPACE ack-$SERVICE-controller
+
+# Delete the CRDs
+kubectl delete -f $CHART_EXPORT_PATH/ack-$SERVICE-controller/crds
```

Comment:
This line actually does exactly what the note above warns about - it uninstalls all CRDs (including the common ones). Perhaps we can have deleting CRDs in its own section to prevent users from simply copy-pasting this block and messing up their other controllers.